### DISCUSSION TITLE
[Question]: encrypt saved mp4 video

### DISCUSSION THREAD

**Discussion by trdb33** [blakeblackshear/frigate#11694] [Ask A Question]
### What is your question:

Hey, I'm using Frigate for several years and I had some questions, let's start with the 1st one
is it possible to encrypt the saved videos ? the idea is to have unusable mp4 files for any user but I can decrypt files in case i need them,
Thanks

**Comment by hawkeye217** [comment]
No, this is not something frigate does natively.
